web/vite.config.js

```diff
@@ -17,6 +18,7 @@ export default defineConfig({
       'xmlhttprequest-ssl': './node_modules/engine.io-client/lib/xmlhttprequest.js',
       // eslint-disable-next-line unicorn/prefer-module
       '@test-data': path.resolve(__dirname, './src/test-data'),
+      '@assets': path.resolve('./src/lib/assets'),
```

Comment:
We chose `$` for alias prefixes in SvelteKit (i.e. `$lib`) because `@` looks like an npm module (it also has more chance of colliding with an npm module added later, but the main reason is just for distinguishing them)

Reply:
Then, it's probably better to keep `$lib/assets`